ITEM.tsv

```diff
@@ -5763,9 +5763,9 @@ ITEM_20150717_005762	Like these types Pearl is a decoration purpose since it is
 ITEM_20150717_005763	Priests who communicated directly with Goddess also don't know lots of parts. However, known up to now can endow a lot of abilities.
 ITEM_20150717_005764	The origin of this bracelet is connecting to Lydia Schaffen and Schaffenstar formed by her. However, There is not restriction to use this bracelet for people who not came from Star Tower.
 ITEM_20150717_005765	Goddess Jurate evicted demons living in the sea from her territory. As Merregina roams on land, its use as material is becoming known. 
-ITEM_20150717_005766	Making the new magical weapons to study strong demons is more important after the Day of Divine Tree.
+ITEM_20150717_005766	Making the new magical weapons to study strong demons is more important after the Medis Diena.
```

Comment:
Maybe:
The study of strong demons and the creation of new magical weapons has become more important after the Medis Diena